### PR TITLE
Introduce new payload -38 to fix a critical hash problem

### PR DESCRIPTION
**Pull request by Jack-Works**
1. Introduce a new -38 payload
2. Apply the stable hash fix on the -38 version

```ts

    // ? payload is 🎼4/4|AESKeyEncrypted|iv|encryptedText|signature|publicShared:||
    // ? if publicShared is true, that means AESKeyEncrypted is shared with public
```

### CODE REVIEW

File: src/components/InjectedComponents/PostInspector.tsx [hunk 1]
```diff
@@ -57,7 +57,11 @@ export function PostInspector(props: PostInspectorProps) {
 
     if (type.encryptedPost) {
         if (!isDebugging) props.needZip()
-        const { iv, ownersAESKeyEncrypted, version } = type.encryptedPost
+        const { iv, version } = type.encryptedPost
+        const ownersAESKeyEncrypted =
+            type.encryptedPost.version === -38
```
Comment (Misaka-0x447f) [inline src/components/InjectedComponents/PostInspector.tsx]: condition may need to be greater than or equal -38.

**Comment by Misaka-0x447f** [comment]
We may need a payload version managing system just like the social-network one.

**Comment by Tedko** [comment]
+1

**Comment by Jack-Works** [comment]
> We may need a payload version managing system just like the social-network one, or the difficult of maintenance may increase quickly.

We do have one

**Comment by Misaka-0x447f** [comment]
> 
> 
> > We may need a payload version managing system just like the social-network one, or the difficult of maintenance may increase quickly.
> 
> We do have one

We do not have one. I means that we need something like ```definePayloadVersion``` so that we will not need to care versioning outside of that file/folder.

**Pull request merged**